src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -1721,7 +1721,9 @@ class spell_gen_cannibalize : public SpellScriptLoader
             // search for nearby enemy corpse in range
             acore::AnyDeadUnitSpellTargetInRangeCheck check(caster, max_range, GetSpellInfo(), TARGET_CHECK_CORPSE);
             acore::WorldObjectSearcher<acore::AnyDeadUnitSpellTargetInRangeCheck> searcher(caster, result, check);
-            caster->GetMap()->VisitFirstFound(caster->m_positionX, caster->m_positionY, max_range, searcher);
+            Cell::VisitWorldObjects(caster, searcher, max_range);
+            if (!result)
```

Comment:
@FrancescoBorzi He's not going to be happy.

Reply:
yikes, will fix all of them

Reply:
:D

Reply:
He didnt see :o